### PR TITLE
map wrong answer reasons to individual cards.

### PR DESCRIPTION
**Pull request by lucahendicott**
Map wrong answer reasons to individual cards.
TODO: update carousel to swipe through each card

### CODE REVIEW

File: mobile/ios/Podfile.lock [hunk 1]
```diff
@@ -381,7 +381,7 @@ PODS:
     - React
   - rn-fetch-blob (0.12.0):
     - React-Core
-  - RNCAsyncStorage (1.17.9):
```
Comment (maniramezan) [inline mobile/ios/Podfile.lock:384]: Looks like you're updating this dependency without updating the version in `package.json`

File: mobile/src/screens/JoinGame/index.js [hunk 2]
```diff
@@ -48,13 +47,20 @@ export default function JoinGame({
...
                     teamMember,
                 })
 
+            case GameSessionState.PHASE_1_RESULTS:
+            case GameSessionState.PHASE_2_RESULTS:
+                return navigation.navigate("PhaseResult", {
+                    gameSession,
+                    team: gameSession.teams.find((team) => team.id === teamId),
+                    teamMember,
+                })
```
Comment (maniramezan) [inline mobile/src/screens/JoinGame/index.js:62]: We'd also need avatars from select team screen to be passed along to all screens

Comment (lucahendicott) [reply mobile/src/screens/JoinGame/index.js:62]: Got it. Will do.

File: mobile/ios/Podfile.lock (hunk 1 above)
Comment (lucahendicott) [reply mobile/ios/Podfile.lock:384]: Oh ok I think I see it. Will update

**Pull request merged**